hr_timesheet_sheet/__manifest__.py

```diff
@@ -26,10 +26,4 @@
         "views/account_analytic_line_views.xml",
         "views/res_config_settings_views.xml",
     ],
-    "assets": {
```

Comment:
Does this mean, it is working from the `web_widget_x2many_2d_matrix` itself which was customized in those js and scss

Reply:
Yes, this didn't worked in 15.0 either btw.